e2-studio-star-rx72n-firmware/src/boot/r_bsp/r_rx_intrinsic_functions.c

```
@@ -0,0 +1,1277 @@
+  while (count > 1) {
+    __builtin_rx_maclo(*ldata1, *ldata2);
+    __builtin_rx_machi(*ldata1, *ldata2);
+    ldata1++;
+    ldata2++;
+    count -= 2;
+  }
+  if (count != 0) {
+    __builtin_rx_maclo(*(const short*)ldata1, *(const short*)ldata2);
+  }
+  __builtin_rx_racw(1);
+  return __builtin_rx_mvfachi();
```

Clang (14.0.6)

[warning] narrowing conversion from 'int' to signed type 'short' is implementation-defined

(bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions)